source/gravity_model/function.cc

```diff
@@ -66,6 +68,24 @@ namespace aspect
       {
         prm.enter_subsection("Function");
         {
+          /**
+           * Choose the coordinates to evaluate the maximum refinement level
+           * function. The function can be declared in dependence of depth,
+           * cartesian coordinates or spherical coordinates. Note that the order
+           * of spherical coordinates is r,phi,theta and not r,theta,phi, since
+           * this allows for dimension independent expressions.
+           */
```

Comment:
Remove all this. The `declare_entry` function already receives a lot of documentation about what this parameter means, so this comment is duplicating information.